base/src/main/scala/co/uproot/abandon/Report.scala

```diff
@@ -252,6 +252,9 @@ object Reports {
 
   def xmlBalanceExport(state: AppState, exportSettings: XmlExportSettings): xml.Node = {
     <abandon>
+      <version>
+        { BuildInfo.version }
+      </version>
```

Comment:
If it's a leaf node, then it is better to not have spaces. It will be easy for the application which is reading the XML file to parse the text content.

Actually, you could even make `version` an attribute of the `abandon` node. No need for a separate node.

If a node is a "structural node" with child nodes, then spacing and indentation makes sense.